site/index.html

```diff
@@ -17,7 +17,6 @@
   <meta property="og:type" content="website" />
   <meta property="csrf-token" content="{{ .CSRF.Token }}" />
   <meta property="build-info" content="{{ .BuildInfo }}" />
-  <meta property="user" content="{{ .User }}" />
```

Comment:
Can I get a gut check here that this is safe to take out?? Seems fine but I am not very familiar with these tags and I don't want to break anything.

Reply:
This shouldn't be taken out! It makes the page load faster. See: https://github.com/coder/coder/blob/21f87313bd5e28dab227ae2cf77d216fe546be10/site/site.go#L302-L307

Reply:
@kylecarbs  @BrunoQuaresma  Gotcha! So if I understand correctly, we return some API responses when the user requests `index.html`. This is achieved via these `meta` tags.

I see this pattern some places in the app: 
`getMetadataAsJSON<Entitlements>("entitlements") ?? API.getEntitlements()`

And I'm confused because 1) wouldn't we always want fresh entitlement data? and 2) don't we already [load entitlement data](https://github.com/coder/coder/blob/main/site/src/components/Dashboard/DashboardProvider.tsx#L90) _again_ in our `DashboardProvider` which wraps all authenticated routes?

Reply:
Ok, but given this pattern:
`async () =>
      getMetadataAsJSON<User>("user") ?? API.getAuthenticatedUser(),`
if a user doesn't refresh the page between login and logout, I think we'll see this bug because the user request is never rehydrated, right